cg/src/domains/processors/processors.requests.js

```diff
@@ -14,6 +14,15 @@ async function getDataForSubject(subjectId) {
   });
 }
 
+async function getRestrictionsForSubject(subjectId) {
+  return await fetch(`${CONTROLLER_URL}/api/processors/subject/${subjectId}/get-restrictions`, {
+    method: 'GET',
```

Comment:
it's not GET by default?

Reply:
It is but clarity is always good. We're building a back-end framework for other devs afterall

Reply:
I see but the GET as default is fairly common and we should have a API documentation somewhere to make it even clearer.